### DISCUSSION TITLE
Error when unlocking the directory, the folder is completely empty.

### DISCUSSION THREAD

**Discussion by GatoOscuro** [mpobaschnig/vaults#73] [General]
**Steps to reproduce**

I was unlocking the vault as usual, when I put the password and access I find that the folder is empty.

**Current behavior**

Currently I can't find a way to recover any of my data that was in the vault, although by analysing the disk I can see an encrypted folder with encrypted information, but I can't import it into Vaults.

**Expected behavior**

Being able to access information.

**Version information**

- Application: Vaults. 
- CryFS (if related): Yes.
- Distribution: LMDE 6.

**Additional information**

Graphical screenshots are attached.
![Captura de pantalla de 2023-11-12 22-22-26](https://github.com/mpobaschnig/vaults/assets/80850364/b9788dae-adb3-4bbb-a7d9-26d9950c7438)
![Captura de pantalla de 2023-11-12 22-21-50](https://github.com/mpobaschnig/vaults/assets/80850364/1b5a86af-9928-4685-918c-f594686fa094)
![Captura de pantalla de 2023-11-12 22-21-29](https://github.com/mpobaschnig/vaults/assets/80850364/b8d2b3db-1fbf-4b45-84a3-34160f8968e0)

**Comment by mpobaschnig** [comment]
Hey, do you have a `cryfs.config` file within the directory? It contains all the needed information to open the vault. If so, you can try to mount it manually by:
```
cryfs /path/to/data/dir /path/to/mount/dir
```

Edit: It also does not look like a CryFS data directory?

**Comment by GatoOscuro** [comment]
> Oye, ¿tienes un `cryfs.config`archivo dentro del directorio? Contiene toda la información necesaria para abrir la bóveda. Si es así, puedes intentar montarlo manualmente:
> 
> ```
> cryfs /path/to/data/dir /path/to/mount/dir
> ```
> 
> Editar: ¿Tampoco parece un directorio de datos CryFS?

Hello, good morning.
I did find the configuration file, but it seems to be skipped and created a new directory, following the instructions. 

![a](https://github.com/mpobaschnig/vaults/assets/80850364/7635bac8-cbd9-4eef-a1e9-edaa8c61a7e9)

And yes, I already found the precise directory with all the encrypted info, I still can't undo it.
![Captura de pantalla de 2023-11-13 15-06-38](https://github.com/mpobaschnig/vaults/assets/80850364/dc07c023-1a49-4f19-9ec0-a273f79b01eb)

**Reply by mpobaschnig** [reply]
From the picture above, it seems that the decryption worked. Is there printed something else? Does a new mountpoint come up in your file manager? Or can you post the output of (if available):
```
gio mount -l
```
